airflow/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -452,89 +473,69 @@ def __init__(
         if self.config_file:
             raise AirflowException("config_file is not an allowed parameter for the GKEStartPodOperator.")
 
-    @staticmethod
-    @contextmanager
-    def get_gke_config_file(
```

Comment:
The method was changed to use another way of connection: instead of direct creating gcloud command to perform any action, now the ssl certificate and url of the cluster are used to connect to specific cluster. 
Can i add some `deprecation warning` for this method instead?

Reply:
I think good to have a warning but let's see what others think...